TWLight/applications/views.py

```diff
@@ -113,19 +113,27 @@ def get_form_class(self):
         """
         fields = {}
         field_order = []
+	open_apps = Application.objects.filter(
+		editor = self.request.user.editor,
+		status__in = (Application.SENT, Application.QUESTION, Application.PENDING)
+	)
+	open_apps_list = []
```

Comment:
`open_apps_list` is a slightly confusing name for this variable, something like `open_app_partners` would be clearer when reading the code.

Reply:
@Samwalton9 I have made the suggested changes, please review them.